docs/pages/pageheader.js

```diff
@@ -397,43 +591,98 @@ function Example() {
+    ],
+  }}
+  secondaryAction={{
+    component: <Button size="lg" text="Analysen anzeigen" />,
+    dropdownItems: [
+      <Dropdown.Link
+        option={{ value: 'Analysen anzeigen', label: 'Analysen anzeigen' }}
+        href="https://pinterest.com"
+      />,
+    ],
+  }}
+  dropdownAccessibilityLabel="Mehr Optionen"
+/>
+    `}
         />
       </MainSection>
-
       <MainSection name="Variants">
         <MainSection.Subsection
-          description={`PageHeader supports an optional \`primaryAction\`. It can be a [Button](/button), a [Link](/link) or an [IconButton](/iconbutton) with a [Tooltip](/tooltip) and optional [Dropdown](/dropdown). Any Buttons or IconButtons should be \`size="lg"\`.`}
+          description={`PageHeader's \`title\` is the main part of the component as it represents the page's main heading (it will always be a level 1 heading). It can be complemented with three additional elements: a thumbnail (left) and a badge and/or a helperIcon (right). Don't forget to localized its content.`}
+          title="Title"
+        >
+          <MainSection.Card
+            shaded
+            cardSize="lg"
+            defaultCode={`
+<PageHeader
+  title="Pinterest app"
+  badge={{ text: 'New', tooltipText: 'New integration' }}
+  helperIconButton={{
+        accessibilityLabel: 'Read more information about the new Pinterest integration',
+        onClick: () => {},
+      }}
+  thumbnail={
+        <Image
+          alt="square"
+          fit="cover"
+          naturalHeight={1}
+          naturalWidth={1}
+          src="https://i.ibb.co/LQc8ynn/image.png"
+        />
+      }
+
+/>
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description={`PageHeader supports an optional \`primaryAction\`. It can be a [Button](/button), a [Link](/link) or an [IconButton](/iconbutton) with a [Tooltip](/tooltip) and optional [Dropdown](/dropdown). Any Buttons or IconButtons should be \`size="lg"\`.
+
+If there's already a primary action elsewhere on the page, PageHeader can have 1 or 2 secondary actions. Use \`primaryAction\` as an additional secondary action.
+
+Primary and secondary actions are consolidated into [Dropdown](https://gestalt.netlify.app/dropdown) below the [sm breakpoint](https://gestalt.netlify.app/screen_sizes#Web-(px)). \`primaryAction\` takes both the main component and its equivalent using Dropdown subcomponents.
+
+For example, Button should be complemented with [Dropdown.Item](https://gestalt.pinterest.systems/dropdown#Dropdown.Item), Link should be complemented with [Dropdown.Link](https://gestalt.pinterest.systems/dropdown#Dropdown.Link), and an IconButton displaying a Dropdown should reuse the same Dropdown subcomponents. Don't forget to pass \`dropdownAccessibilityLabel\` for the IconButton consolidating all actions into [Dropdown](https://gestalt.netlify.app/dropdown) below the sm breakpoint.
+`}
```

Comment:
I think it'd be helpful to add more line here: "Resize your window to observe how the PageHeaders below adapt to smaller screen widths".

Reply:
DONE